tox.ini

```diff
@@ -5,15 +5,13 @@
 
 [tox]
 envlist =
-    py2{6,7}-django{14,16},
-    py27-django{17,18},
+    py27-django{18,111},
     # py34-django{17,18}
 
 [testenv]
-commands = nosetests
+commands = {envpython} run_tests.py
 deps =
     nose
-    django14: Django>=1.4, <1.5
-    django16: Django>=1.6, <1.7
-    django17: Django>=1.7, <1.8
```

Comment:
are the previous django version failing to pass the tests? or you just dont want to have to bother with them?

Reply:
I don't want to bother with them. They are not supported any longer anyway.